### PR TITLE
Add flask-multistatic

### PR DESCRIPTION
**Pull request by AdMax18**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/flask-multistatic/LICENSE [hunk 1]
```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```
Comment (beenje) [inline recipes/flask-multistatic/LICENSE:1]: This file is part of the python package. No need to add it in this repo.

File: recipes/flask-multistatic/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+about:
+  home: https://pagure.io/flask-multistatic
+  summary: Simple flask plugin to allow overriding static files
+  license: GPL-3.0-or-later
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - mxr-conda
```
Comment (beenje) [inline recipes/flask-multistatic/meta.yaml:42]: Could someone from @mxr-conda please confirm they want to be listed as maintainer to this feedstock?

Comment (mxr-conda) [reply recipes/flask-multistatic/meta.yaml:42]: @beenje Agree to maintaain

**Comment by AdMax18** [comment]
closed because fork and branch were deleted - opened new with changes: https://github.com/conda-forge/staged-recipes/pull/17043

**Pull request closed without merging**